actors/builtin/verifreg/verified_registry_actor.go

```diff
@@ -119,6 +131,13 @@ func (a Actor) AddVerifiedClient(rt vmr.Runtime, params *AddVerifiedClientParams
 			rt.Abortf(exitcode.ErrNotFound, "Invalid verifier %v", verifierAddr)
 		}
 
+		// Validate client to be added isn't a verifier
+		found, err = verifiers.Get(AddrKey(params.Address), nil)
+		builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to get verifier")
+		if found {
+			rt.Abortf(exitcode.ErrIllegalArgument, "verifier %v cannot be added as a verified client", params.Address)
+		}
```

Comment:
Blocking: unless we also constrain AddVerifier to prevent existing clients, we won't have an invariant.

Reply:
This is done.

Reply:
This has been done.